html/lesson7/tutorial.md

```diff
@@ -0,0 +1,420 @@
+
+HTML5 introduced a method to let web developers take control over the viewport, using the <meta> tag.
+
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+
+width=device-width sets the width of the page to be the screen-width of the device.
+
+initial-scale=1.0 sets the initial zoom level when the page is first loaded by the browser.
+
+Add the meta viewport tag to your html, under your css links.
+
+
+##Mobile First Design
+
+Historically, web designers have tended to design from the biggest screen down to the smallest. The “main” design, with the most functionality, would be for the full desktop view.
+
+The way we consume content on the web has changed a lot and more and more internet traffic is from 'mobile only' users. The previous approach will cause these users problems, because the priority is given to desktop users. Design elements and functionality that look and work great on desktop sometimes do not translate to a good mobile experience, and at worst can make your site unusable for mobile users.
+
+Mobile first design takes the opposite approach. The design process is started with consideration for mobile constraints and then you work your way up from there.
+
+<img src="assets/images/respond.png" style="max-width:500px">
+
+The mobile first approach aims to provide the best possible experience for mobile users by focussing on core content and functionality and by giving consideration to user's available bandwidth and likely needs.
+
+
+###Structure your content first
+
+The content of your site is the important stuff! Think about what you want your users to see and do when they get to your site. Create a hierachy of content that will get your information across to your users in a way that is meaningful and cohesive.
+
+In the case of our example, in order of importance, we've got the site name, the text, a list of useful offsite resources and info about the author. If we were to show these things on a mobile they'd appear in this order eg.
+
+![](assets/images/mobilelayout.png)
+
+So, lets get some markup written!  We're going to use our html5 elements to lay out the site so, put in a header, a section, a sidebar div and a footer with relevant class names
+
+    <header class="header">
+    </header>
+
+    <section class="about">
+    </section>
+
+    <div class="sidebar">
+    </div>
+
+    <footer class="footer">
+    </footer>
+
+This is the structure of our page.
+
+###Add in the content
+
+Now we're ready to flesh out our structure with content.  Looking at the example page, starting from the top we can see that the header contains an image and a heading, so lets put them into our HTML, with class names
+
+    <img class="header_img" src="assets/images/gracehopper_portrait.jpg" />
```

Comment:
can remove the / from the end of the img element please. I removed them all in a previous PR as that is old HTML so to keep it consistent throughout all tutorials.